### ISSUE TITLE
autonomousScoreConeGetConeGetCloseCommand runs the wrong way ( mirrored) when alliance is red

### ISSUE THREAD

**Issue by dcowden** [entech281/Robot2023#255]
With alliance Red, this command works fine:
autonomousScoreConeGetConeScoreMidCommand

but this one tries to run the  blue version ( it's backwards):
autonomousScoreConeGetConeGetCloseCommand

**Comment by dcowden** [comment]
this actually resulted in running into the barrier, unfortuately

**Comment by dcowden** [comment]
attempted fix by adding this function:

```    private double getAutoConePickupTurnAngle() {
        if ( DriverStation.getAlliance() == Alliance.Red) {
        	return -140.0;
        }
        else {
        	return 140.0;
        }    	
    }```

fortunately this bug proves that my fix for #256 will work, so i think the code in dc_get_it_working_again branch will fix this
